### ISSUE TITLE
Angular 4 DatePipe format not working properly for minutes and seconds in IE & Edge

### ISSUE THREAD

**Issue by firstor** [first87/polymer-chat#6]
#### Reproduction
 - Angular 4.2.3
 - IE 11, Edge 14
 - Import Intl polyfill (according to [Angular Optional browser features to polyfill](https://angular.io/guide/browser-support#optional-browser-features-to-polyfill))
   ```ts
   /* ========= src/polyfills.ts ========= */
   ...
   /**
    * Date, currency, decimal and percent pipes.
    * Needed for: All but Chrome, Firefox, Edge, IE11 and Safari 10
    */
   import 'intl';  // Run `npm install --save intl`.
   /**
    * Need to import at least one locale-data with intl.
    */
   import 'intl/locale-data/jsonp/en-US';
   ```
 - Use DatePipe to format hours and minutes with `HH:mm` (according to [DatePipe format](https://angular.io/api/common/DatePipe))
   ```html
   <!-- ========= message-item.component.html ========= -->
   ...
    <span class="posted">{{message.posted | date:'HH:mm'}}</span>
   ...
   ```

#### Expected output vs Current output
 - `2017-06-12 09:03:10` => `09:03` but `09:00:6/12/2017 9:03:10 AM`
 - `2017-06-12 13:23:30` => `13:23` but `13:00:6/12/2017 1:23:30 PM`

**Comment by firstor** [comment]
Similar issues were reported:
 - https://github.com/angular/angular/issues/9524
 - https://github.com/angular/angular/issues/10147

**Comment by firstor** [comment]
Seems that https://github.com/angular/angular/issues/9524#issuecomment-254810775 is the best solution right now!

**Comment by firstor** [comment]
Fixed with https://github.com/firstor/polymer-chat/pull/7

**Comment by firstor** [comment]
Also fixed with #8
